index.js

```diff
@@ -214,110 +214,102 @@ const normalizeRetryOptions = retry => {
 
 class Ky {
 	constructor(input, {
-		timeout = 10000,
 		hooks,
-		throwHttpErrors = true,
-		searchParams,
 		json,
+		onDownloadProgress,
+		prefixUrl,
 		retry = {},
-		...otherOptions
+		searchParams,
+		throwHttpErrors = true,
+		timeout = 10000,
+		...fetchOptions
 	}) {
 		this._retryCount = 0;
-
+		this._input = input;
 		this._options = {
-			method: 'get',
-			credentials: 'same-origin', // TODO: This can be removed when the spec change is implemented in all browsers. Context: https://www.chromestatus.com/feature/4539473312350208
+			hooks: deepMerge({
+				beforeRequest: [],
+				beforeRetry: [],
+				afterResponse: []
+			}, hooks),
+			json,
+			onDownloadProgress,
+			prefixUrl: String(prefixUrl || ''),
 			retry: normalizeRetryOptions(retry),
-			...otherOptions
+			searchParams,
+			throwHttpErrors,
+			timeout
+		};
+		this._fetchOptions = {
+			// TODO: credentials can be removed when the spec change is implemented in all browsers. Context: https://www.chromestatus.com/feature/4539473312350208
```

Comment:
I tested Chrome, Firefox, Safari, and Edge 18, and all of them default to `same-origin` now. Not sure about mobile browsers, though. Should we remove this?

Reply:
Nah, not worth the risk. It's only a few bytes anyway.